src/components/Icon/IconProject.test.js

```diff
@@ -1,8 +1,17 @@
-/* global describe */
+/* global describe, expect, it, shallow */
+import React from 'react';
 import * as common from '../../../test/unit/commonTests';
 import IconProject from './IconProject';
 
 describe('IconProject', () => {
     common.isConformant(IconProject, { tagName: 'svg' });
     common.addsClassName(IconProject, 'uir-icon');
+
+    it('sets title', () => {
+        const wrapper = shallow((
+            <IconProject title="Test title" />
+        ));
```

Comment:
Don't really care, but just so it doesn't go unnoticed there's an extra `(` here.

Reply:
It's intentional. Otherwise the linter would complain about the lack of comma after the function argument. Using an additional set of `()` allows us to omit the comma after JSX.